app/adapters/v3.js

```diff
@@ -1,3 +1,5 @@
+import { underscore } from '@ember/string';
+import { get } from '@ember/object';
 import Ember from 'ember';
```

Comment:
It looks to me that this is for `Ember.String.pluralize`, is it an oversight?

I searched the branch and many of the remaining imports are for `Ember.testing` which I assume is not possible to address yet, I don’t know too much about this… I also see `Ember.Handlebars.Utils`, `Ember.Inflector.inflector`, `Ember.merge`…

Is there an advantage to a partial conversion in this direction? Is it worth waiting until we can get rid of all our monolithic imports?

This has turned into a more meta comment than this single line hehe

Reply:
@backspace So, I was surprised to discover that you can't import `import { pluralize } from '@ember/string'`, likely because `pluralize` is actually on `DS.String` and not `Ember.String`. (How does this work lol)

I figured that it didn't hurt to move over in pieces, though I'm fine if we want to wait until everything can be removed. Your call.

Reply:
oh yes, I guess it’s a “prototype extension”, makes sense. Well I don’t see a reason to not move, though I don’t know how I’ll remember to make this change hehe